### ISSUE TITLE
Update /wmg layout to be three panels

### ISSUE THREAD

**Issue by tihuan** [chanzuckerberg/single-cell-data-portal#1847]

**Comment by seve** [comment]
@tihuan Can you attach a mock of some sort? Even if it's just a screenshot of CC's Search/Filter feature.

**Comment by tihuan** [comment]
Morning Seve! Harley is gonna finalize the V1 mock today, but I think the layout is likely staying the same 👌

https://www.figma.com/file/htICFl7zihfs1z9eIYVwzw/Gene-Search?node-id=2402%3A423263

Is that what you're looking for?

Thank you!

**Comment by seve** [comment]
Right! Thank you, forgot you mentioned that yesterday afternoon!

Just to clarify/refine, the layout is more akin to explorer with a main visualization book-ended by two sidebars, yes? I'm being nitpicky just to establish the difference in priority between the two headspaces: three panels with equal thirds of importance, vs main single importance viz with two supplementing sidebars.

**Comment by tihuan** [comment]
Ooh good point that Explorer has the three panel layout too 💡

That's a great question, from the mock it doesn't seem like three panels will be equal in size, but they will be collapsible! 

Let's confirm with @hthomas-czi on the actual specs just in case 🙏

And please feel free to reach out to him directly if he doesn't respond here, since I know some people don't check GitHub notif frequently 😄

Thanks a lot!

**Comment by hthomas-czi** [comment]
@seve Yes, that's the right way to think about the interface (main workspace + 2 sidebars). It differs from Explorer in the ways @tihuan described — namely, they're asymmetrically and collapsible.

**Comment by tihuan** [comment]
Hey @seve ! 

Harley finalized the panel layout in the [latest Figma](https://www.figma.com/file/htICFl7zihfs1z9eIYVwzw/Gene-Search?node-id=2402%3A423263) 🚀

What we discussed in the kickoff meeting this morning is that the left panel expand/collapse will reposition the middle panel (instead of an overlay), but won't resize the middle panel. And the right panel expand/collapse will overlay the middle panel, so it could cover the middle panel partially when expanded and that's fine

Hope this makes sense! 

Thank you!

**Comment by tihuan** [comment]
I just updated the DoD, please feel free to edit it as you see fit! Thanks!

**Comment by tihuan** [comment]
@seve also just to be super explicit, the three panel layout doesn't include contents in the panel! We have other tickets for those 👍

Please feel free to check https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/1844 for all the tickets and help see if I missed anything 😆 Thank you!

**Comment by tihuan** [comment]
confirmed in staging!

![Screen Shot 2022-02-17 at 12.19.36 PM.png](https://images.zenhubusercontent.com/5ebed8780d2d150fc281ad55/e71d3533-ed66-4548-9037-85fdb4b29a86)
